src/ui/components/edit_user_pop_up/edit_user_pop_up.py

```diff
@@ -1,170 +1,110 @@
+            "user_type_button": ("xpath", "//input[@value='ROLE_USER']/../.."),
+            "manager_icon": ("xpath", "./descendant::div[@class='ellipse'][2]"),
+            "manager_type_button": ("xpath", "//input[@value='ROLE_MANAGER']/../.."),
+            "last_name_input_element": ("xpath", "./descendant::div[contains(@class,'user-edit-input')][1]"),
+            "first_name_input_element": ("xpath", "./descendant::div[contains(@class,'user-edit-input')][2]"),
+            "phone_input_element": ("xpath", "./descendant::div[contains(@class,'user-edit-input')][3]"),
+            "phone_country_code": ("xpath", ".//span[@class='ant-input-group-addon']"),
+            "email_input_element": ("xpath", "./descendant::div[contains(@class,'user-edit-input')][4]"),
+            "upload_user_photo_element": ("xpath", ".//span[contains(@class, 'ant-upload-wrapper')]"),
+            "upload_user_photo_input": ("xpath", ".//input[@id='edit_urlLogo']"),
+            "photo_title": ("xpath", ".//label[@for='edit_urlLogo']"),
+            "question_circle_for_photo": ("xpath", ".//span[@aria-label='question-circle']"),
+            "photo_tooltip_form": ("xpath", "//div[@class='ant-tooltip-inner']"),
+            "checkbox_change_password_title": ("xpath", ".//div[contains(@class, 'align-checkbox')]/text"),
+            "checkbox_change_password_input": ("xpath", ".//input[@name='checkbox']"),
+            "current_password_input_element": ("xpath", "(//div[contains(@class,'item-control-input')]"
+                                                        "/span[contains(@class,'ant-input-password')"
+                                                        " and not (contains(@class,'login-box'))])[1]"),
+            "new_password_input_element": ("xpath", "(//div[contains(@class,'item-control-input')]"
+                                                    "/span[contains(@class,'ant-input-password')"
+                                                    " and not (contains(@class,'login-box'))])[2]"),
+            "confirm_password_input_element": ("xpath", "(//div[contains(@class,'item-control-input')]"
+                                                        "/span[contains(@class,'ant-input-password')"
+                                                        " and not (contains(@class,'login-box'))])[3]"),
+            "submit_button": ("xpath", ".//button[contains(@class, 'submit-button')]")
+        }
+        self._wait = WebDriverWait(self.driver, 30)
+        self._actions = ActionChains(self.driver)
+
+    def click_on_manager_type_button(self) -> Self:
+        return self.manager_type_button.click
 
     def click_on_user_type_button(self):
-        return self.user_type_button.click()
-
-    @property
-    def user_icon(self) -> WebElement:
-        if not self._user_icon:
-            self._user_icon = self.node.find_element(*USER_ICON)
-        return self._user_icon
+        return self.user_type_button.click
```

Comment:
Probably you missed () after click method in this row and other methods below this.
It is better to use PageFactory click_button() method, it waits element becomes clickable.

Reply:
Rewrote